src/index.js

```diff
@@ -185,6 +228,8 @@ AFRAME.registerComponent('streetmix-loader', {
           JSON.stringify({ streetmixSegmentsMetric: streetmixSegments })
         );
         el.emit('streetmix-loader-street-loaded');
+        // the streetmix data has been loaded, set the synchronized flag
+        data.synchronized = true;
```

Comment:
Don't set it like that, otherwise the change won't be properly exported. Use
`this.el.setAttribute('street', 'synchronized', true}`

Reply:
Ok, I'll do tis way. But I tested export and import and it works well

Reply:
The current code may work on aframe 1.5.0 but I think it won't work on aframe 1.6.0 or master because of internals restructuring, calling getAttribute('street') won't return the modified data object.

Reply:
Thanks for clarifying. Yes, I have read the documentation why it is better to use setAttribute. But in general it would be better to use an internal variable of the component in these cases, which is saved in JSON, but will not be displayed in the editor. There is no such mechanism in the component implementation, but it could be done using __innerAttrName and internal get/set functions

Reply:
@Algorush it's ok for this variable to be exposed to the end-user